--- a/src/modules/aws.rs	1969-12-31 20:00:01.000000000 -0400
+++ b/src/modules/aws.rs	2024-07-22 12:53:06.598525330 -0300
@@ -141,7 +141,7 @@
         let creds = get_creds(context, aws_creds)?;
         let section = get_profile_creds(creds, aws_profile)?;
 
-        let expiration_keys = ["expiration", "x_security_token_expires"];
+        let expiration_keys = ["aws_expiration", "expiration", "x_security_token_expires"];
         expiration_keys
             .iter()
             .find_map(|expiration_key| section.get(expiration_key))
